resources/smcp-templates/v2.1/external

```diff
@@ -0,0 +1,107 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: external
+spec:
+  version: v2.1
+  gateways:
+    enabled: false
+    ingress:
+      enabled: false
+      ingress: false
+      service:
+        type: ClusterIP
+    egress:
+      enabled: false
+    openshiftRoute:
+      enabled: false
+
+  general:
+    logging:
+      componentLevels:
+        default: warn
+
+  proxy:
+    injection:
+      autoInject: false
+    networking:
+      protocol:
+        autoDetect:
+          inbound: false
+          outbound: false
+      dns:
+        refreshRate: 300s
+
+  security:
+    enabled: false
+    identity:
+      type: Kubernetes
+
+  telemetry:
+    enabled: false
+    type: Istiod
+
+  controlPlaneMode:
+    enabled: true
+
+  remoteMode:
+    enabled: false
+
+  cni:
```

Comment:
I don't believe this field exists in smcp.spec.

Reply:
DId you mean only the cni parameter?

Reply:
I will remove this, thank you